deployments/fargate/README.md

```diff
@@ -0,0 +1,243 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, the
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed as a **sidecar** (additional container) to ECS tasks.
+
+Requires Connector release v0.32.0 or newer which corresponds to image tag 0.32.0 and newer.
+See image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+## Getting Started
+Copy the default Collector container definition JSON below. Replace `MY_SPLUNK_ACCESS_TOKEN`
+and `MY_SPLUNK_REALM` with valid values. Update the image tag to the newest
+version then add the JSON to the `containerDefinitions` section of your task definition
+JSON.
+```json
+{
+  "portMappings": [
+    {
+      "hostPort": 13133,
+      "protocol": "tcp",
+      "containerPort": 13133
+    },
+    {
+      "hostPort": 6060,
+      "protocol": "tcp",
+      "containerPort": 6060
+    },
+    {
+      "hostPort": 14250,
+      "protocol": "tcp",
+      "containerPort": 14250
+    },
+    {
+      "hostPort": 9943,
+      "protocol": "tcp",
+      "containerPort": 9943
+    },
+    {
+      "hostPort": 9411,
+      "protocol": "tcp",
+      "containerPort": 9411
+    },
+    {
+      "hostPort": 9080,
+      "protocol": "tcp",
+      "containerPort": 9080
+    }
+  ],
```

Comment:
I don't think this is required. By default fargate network is shared between containers, so we should be able to send traces to localhost from other containers. I tested tracing, it works well

Reply:
Ok, I'll remove the port mappings.